spec/datadog/core/workers/runtime_metrics_spec.rb

```diff
@@ -282,7 +282,11 @@
       it 'produces metrics every interval' do
         worker.perform
 
-        sleep(default_flush_interval + 0.2)
+        Thread.pass # Let the background thread start
+
+        sleep(default_flush_interval)
+
+        Thread.pass # Let the background thread run, if it hasn't during the sleep above
```

Comment:
I'm trying something a bit different here: adding `Thread.pass` when the test wants the background job to run.
In this case, we want it to run once immediately in the background, then after the sleep internal, run again.